doc/rtd/reference/yaml_examples/update_upgrade.rst

```diff
@@ -0,0 +1,35 @@
+.. _cce-update-upgrade:
+
+Update, upgrade and install packages
+************************************
+
+These examples show you how to install, update, and upgrade packages during
+boot. By default, this is set to "none", but if any packages are specified
+then ``package_update`` will be set to ``true``.
+
+Packages can be supplied as either a single package name, or as a list with
```

Comment:
The supported format is more complex than this now. I would just nuke this paragraph.

Reply:
Done :)